src/privileged/popupNotification/api.js

```diff
@@ -14,7 +14,7 @@ var {EventManager, EventEmitter} = ExtensionCommon;
 var {Management: {global: {tabTracker}}} = ChromeUtils.import("resource://gre/modules/Extension.jsm", {});
 
 // eslint-disable-next-line no-undef
-XPCOMUtils.defineLazyModuleGetter(
+ChromeUtils.defineModuleGetter(
```

Comment:
These two changes were due to the updated **eslint-plugin-mozilla** rules:

```
/Users/pdehaan/dev/github/mozilla/CookieRestrictionsShield/src/privileged/popupNotification/api.js
  17:1  error  Please use ChromeUtils.defineModuleGetter instead of XPCOMUtils.defineLazyModuleGetter  mozilla/use-chromeutils-import

/Users/pdehaan/dev/github/mozilla/CookieRestrictionsShield/src/privileged/trackers/api.js
  12:1  error  Please use ChromeUtils.defineModuleGetter instead of XPCOMUtils.defineLazyModuleGetter  mozilla/use-chromeutils-import
```